packages/db/lib/seed.mjs

```diff
@@ -55,8 +51,19 @@ async function seed (_args) {
       await migrator.close()
     }
   }
-
-  await execute(logger, args, config)
+  let seedFile = args._[0]
+  if (!seedFile) {
+    throw new errors.MissingSeedFileError()
+  }
+  // check if we are in Typescript and, in case, compile it
+  if (seedFile.endsWith('.ts')) {
+    await tsCompiler.compile(process.cwd(), configManager.current, logger)
```

Comment:
@mcollina I was thinking I could use `dirname(configManager.fullPath)` as `cwd` WDYT?

Reply:
seems good for now